pkgs/by-name/dp/dpp/package.nix

```
@@ -0,0 +1,34 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, cmake
+, pkg-config
+, zlib
+, openssl
+, libsodium
+, libopus
+}:
+
+stdenv.mkDerivation rec {
+  pname = "DPP";
+  version = "10.0.26";
+
+  src = fetchFromGitHub {
+    owner = "brainboxdotcc";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-o78ijctDqrONyi8A3+FXvnK9q97B4j1ZIQYNUgl6XJU=";
+  };
+
+  nativeBuildInputs = [ cmake pkg-config ];
+
+  buildInputs = [ zlib openssl libsodium libopus ];
+
+  meta = with lib; {
+    description = "C++ Discord API Bot Library - D++ is Lightweight and scalable for small and huge bots!";
```

```suggestion
    description = "Lightweight and efficient library for Discord written in modern C++";
```

That's a better description